README.md

```
@@ -61,6 +61,14 @@ mxlrcsvc-go "Dream Theater"
 
 > **_The `-d/--depth` argument limit the depth of subdirectory to scan. Use `-d 0` or `--depth 0` to only scan the specified directory._**
 
+## Development
+
+Run the lightweight CLI smoke test:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)